### ISSUE TITLE
2.x: org.eclipse.microprofile.rest.client.tck.timeout tests failing on 2.x branch

### ISSUE THREAD

**Issue by barchetta** [oracle/helidon#4763]
## Environment Details
* Helidon Version: 2.5.3-SNAPSHOT

----------

## Problem Description

Some `org.eclipse.microprofile.rest.client.tck.timeout` tests have started failing on helidon-2.x pipeline.  I've seen this for multiple PRs.  Here is one example:

```
13:50:43,835 [ERROR] Tests run: 224, Failures: 4, Errors: 0, Skipped: 13, Time elapsed: 425.108 s <<< FAILURE! - in TestSuite
13:50:43,836 [ERROR] org.eclipse.microprofile.rest.client.tck.timeout.TimeoutViaMPConfigTest.testConnectTimeout  Time elapsed: 0.043 s  <<< FAILURE!
org.testng.TestException: 

Expected exception of type class javax.ws.rs.ProcessingException but got java.lang.AssertionError: expected [true] but found [false]
	at org.testng.internal.ExpectedExceptionsHolder.wrongException(ExpectedExceptionsHolder.java:68)
	at org.testng.internal.Invoker.considerExceptions(Invoker.java:1120)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:603)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.runSuites(TestNG.java:1028)
	at org.testng.TestNG.run(TestNG.java:996)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertTrue(Assert.java:44)
	at org.testng.Assert.assertTrue(Assert.java:54)
	at org.eclipse.microprofile.rest.client.tck.timeout.TimeoutViaMPConfigTest.checkTimeElapsed(TimeoutViaMPConfigTest.java:69)
	at org.eclipse.microprofile.rest.client.tck.timeout.TimeoutTestBase.testConnectTimeout(TimeoutTestBase.java:68)
	...
```

**Comment by barchetta** [comment]
In terms of timing, these failures starting happening around the time this PR was merged: #4745

**Comment by barchetta** [comment]
These tests were already excluded in 3.x/4.x. Added exclusions in 2.x.
